octobot_trading/personal_data/portfolios/portfolio_value_holder.py

```diff
@@ -305,7 +389,12 @@ def _try_to_ask_ticker_missing_symbol_data(self, currency, symbol, reversed_symb
         elif self.portfolio_manager.exchange_manager.symbol_exists(reversed_symbol):
             symbols_to_add = [reversed_symbol]
 
-        new_symbols_to_add = [s for s in symbols_to_add if s not in self.initializing_symbol_prices_pairs]
+        new_symbols_to_add = [
+            s
+            for s in symbols_to_add
+            if s not in self.initializing_symbol_prices_pairs
+            and s not in self.portfolio_manager.exchange_manager.exchange_config.traded_symbol_pairs
```

Comment:
Can you explain why we should add this condition please ?

Reply:
add to comments